.env.production

```diff
@@ -17,3 +17,4 @@ SENTRY_ORG=plant-for-the-planet
 SENTRY_PROJECT=planet-donations
 SENTRY_AUTH_TOKEN=
 SOURCE_VERSION=
+RECURRENCY=true
```

Comment:
remove this value

Reply:
@tejassonar why would you remove recurrent donations? We would need tests for recurring donations as well.

Reply:
This is production env. We do not use production env here, right?